addon/ng2/blueprints/component/index.js

```diff
@@ -1,29 +1,26 @@
-var stringUtils = require('ember-cli/lib/utilities/string');
 var dynamicPathParser = require('../../utilities/dynamic-path-parser');
 
 module.exports = {
   description: '',
 
-  normalizeEntityName: function(entityName) {
+  normalizeEntityName: function (entityName) {
     var parsedPath = dynamicPathParser(this.project, entityName);
-    
+
     this.dynamicPath = parsedPath;
     return parsedPath.name;
   },
 
-  locals: function(options) {
-    return {
-      dynamicPath: this.dynamicPath.dir
-    };
+  locals: function () {
+    return {dynamicPath: this.dynamicPath.dir};
```

Comment:
I added `"object-curly-spacing": [2, "always"]` as a rule and it enforces the following `return { dynamicPath: this.dynamicPath.dir };`
Brace style `"brace-style": [2, "1tbs", { "allowSingleLine": false }]` is also there but that does not affect return objects.